src/Hangfire.LiteDB/LiteDbMonitoringApi.cs

```diff
@@ -107,7 +107,7 @@ public JobDetailsDto JobDetails(string jobId)
                     CreatedAt = x.CreatedAt,
                     Reason = x.Reason,
                     Data = x.Data
-                }).OrderByDescending(x => x.CreatedAt).ToList();
+                }).ToList().OrderByDescending(x => x.CreatedAt).ToList();
```

Comment:
Why `ToList` is used twice?

Reply:
For the operation to be done locally instead of the database